q2_vsearch/plugin_setup.py

```diff
@@ -313,6 +315,8 @@
         'qminout': ('The minimum allowed quality score to use in output.'),
         'qmax': ('The maximum allowed quality score in the input.'),
         'qmaxout': ('The maximum allowed quality score to use in output.'),
+        'threads': ('The number of threads to use for computation. Does '
+                    'not scale much past 4 threads.')
```

Comment:
Is it true that the scaling falls off after 4 (even when you have more cores, say a hexcore or better)?

Reply:
I tested this from a RAM disk /dev/shm/ and the most it was using was ~300% CPU 🤷‍♂

Reply:
works for me!